app/presenters/worldwide_organisation_presenter.rb

```diff
@@ -109,12 +109,14 @@ def corporate_information_pages
     ordered_cips = content_item.dig("details", "ordered_corporate_information_pages")
     return [] if ordered_cips.blank?
 
-    ordered_cips.map do |cip|
-      {
-        text: cip["title"],
-        url: cips.find { |cp| cp["content_id"] == cip["content_id"] }["web_url"],
-      }
+    links = []
+    ordered_cips.each do |cip|
+      text = cip["title"]
+      link = cips.find { |cp| cp["content_id"] == cip["content_id"] }["web_url"]
+      hyperlinked_text = "<a href =#{link} class=govuk-link>#{text}</a>"
+      links << hyperlinked_text
     end
+    links
```

Comment:
Should we follow the pattern for this set higher up in the file, lines 27-35?

Reply:
YES!